### PR TITLE
Update contact information

### PR DESCRIPTION
**Pull request by canyon289**
Making a minor change to the doc to remove reference to a mailing list and group that I doubt anyone here checks

### CODE REVIEW

**Comment by canyon289** [comment]
Having trouble compiling the docs so I don't know exactly what this looks like when compiled but I think its right

File: doc/introduction.txt [hunk 1]
```diff
@@ -211,18 +211,12 @@ Here is the state of that vision as of November 15th, 2017 (after Theano 1.0.0):
...
 
-Discussion about Theano takes place in the theano-dev_ and
-theano-users_ mailing lists. People interested in development of
-Theano should check the former, while the latter is reserved for
-issues that concern the end users.
-
+Discussion about Theano 
 Questions, comments, praise, criticism as well as bug reports should
-be submitted to these mailing lists.
+be submitted in form of an issue ticket tracker located at the following link pymc-theano-dev_
```
Comment (brandonwillard) [inline doc/introduction.txt:216]: "...submitted to the project's issue tracker on GitHub: pymc-theano-dev_"

We should also limit the accepted correspondence to questions and bug reports.

**Review by brandonwillard — CHANGES_REQUESTED** [review]

File: doc/introduction.txt (hunk 1 above)
Comment (canyon289) [reply doc/introduction.txt]: Updated

File: doc/introduction.txt [hunk 2]
```diff
@@ -211,18 +211,12 @@ Here is the state of that vision as of November 15th, 2017 (after Theano 1.0.0):
...
-Discussion about Theano takes place in the theano-dev_ and
-theano-users_ mailing lists. People interested in development of
-Theano should check the former, while the latter is reserved for
-issues that concern the end users.
-
-Questions, comments, praise, criticism as well as bug reports should
-be submitted to these mailing lists.
+Discussion about Theano 
+Questions and bug reports should
+be submitted in form of an issue ticket, tracker located at the following link pymc-theano-dev_
```
Comment (brandonwillard) [inline doc/introduction.txt:216]:
```suggestion
be submitted in the form of an issue at pymc-theano-dev_
```

**Review by brandonwillard — CHANGES_REQUESTED** [review]

File: doc/introduction.txt [hunk 3]
```diff
@@ -211,18 +211,12 @@ Here is the state of that vision as of November 15th, 2017 (after Theano 1.0.0):
...
 
-Discussion about Theano takes place in the theano-dev_ and
-theano-users_ mailing lists. People interested in development of
-Theano should check the former, while the latter is reserved for
-issues that concern the end users.
-
-Questions, comments, praise, criticism as well as bug reports should
-be submitted to these mailing lists.
+Discussion about Theano 
+Questions and bug reports should
```
Comment (brandonwillard) [inline doc/introduction.txt:215]:
```suggestion
Questions and bug reports should
```

**Review by brandonwillard — CHANGES_REQUESTED** [review]

**Comment by canyon289** [comment]
Thank you

**Pull request merged**